examples/dev/helm_values/AlphaMSP/hlf-ord/ord1.yaml

```diff
@@ -8,11 +8,15 @@ persistence:
 ord:
   type: Solo
   mspID: AlphaMSP
-
+  tls:
```

Comment:
I would create a separate dev_tls example with these changes, and leave the basic dev example as it was.